code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -274,11 +274,12 @@ var/const/NO_EMAG_ACT = -50
 
 /obj/item/weapon/card/id/centcom/ERT
 	name = "\improper Emergency Response Team ID"
+	icon_state = "centcom"
 	assignment = "Emergency Response Team"
 
-/obj/item/weapon/card/id/centcom/ERT/New()
+/obj/item/weapon/card/id/ERT/New()
+	access = get_all_accesses() + get_centcom_access("Emergency Response Team")

```

Comment:
This may result in the ID having standard access at the end. Do this:
```
/obj/item/weapon/card/id/ERT/New()
    ..()
    access = get_all_accesses() + get_centcom_access("Emergency Response Team")
```